### PR TITLE
implicit_tls

### PR DESCRIPTION
**Pull request by mamico**

### CODE REVIEW

File: src/Products/MailHost/dtml/manageMailHost.dtml [hunk 1]
```diff
@@ -78,6 +78,22 @@
...
+        Implicit TLS
+      </label>
+      <div class="form-check">
+        <input id="implicit_tls" class="form-check-input" type="checkbox"
+               name="implicit_tls:boolean"
+               <dtml-if "implicit_tls">checked</dtml-if>>
+        <small>
+          Use TLS from the beginning of the connection,
+          known as SMTPS and commonly used on TCP port 465.
+          force_tls and no_tls are ignored if this is set.
```
Comment (dataflake) [inline src/Products/MailHost/dtml/manageMailHost.dtml:92]: There's no `no_tls` in this code:

```suggestion
          force_tls is ignored if this is set.
```

**Review by dataflake — APPROVED** [review]
Don't forget the change log entry

**Comment by mamico** [comment]
I opened a PR on Plone and manually tested different SMTP configurations.

**Review by mauritsvanrees — APPROVED** [review]
LGTM.

Small reminder just in case it is needed: in the `zopefoundation` organisation it is customary to merge your own PR once it has been approved.

**Comment by mauritsvanrees** [comment]
@icemac Could you make a release with this please?

**Comment by dataflake** [comment]
Version 6.0 is now published.

**Comment by mauritsvanrees** [comment]
Thanks!

**Pull request merged**